packages/wonder-blocks-clickable/src/components/__docs__/clickable-behavior.stories.js

```diff
@@ -0,0 +1,92 @@
+// @flow
+import * as React from "react";
+import {StyleSheet} from "aphrodite";
+
+import {getClickableBehavior} from "@khanacademy/wonder-blocks-clickable";
+import {View} from "@khanacademy/wonder-blocks-core";
+import Color from "@khanacademy/wonder-blocks-color";
+import Spacing from "@khanacademy/wonder-blocks-spacing";
+
+import type {StoryComponentType} from "@storybook/react";
+import ComponentInfo from "../../../../../.storybook/components/component-info.js";
+import {name, version} from "../../../package.json";
+import argTypes from "./clickable-behavior.argtypes.js";
+
+const ClickableBehavior: React.ComponentType<
+    React.ElementConfig<typeof ClickableBehavior>,
+> = getClickableBehavior();
+
+export default {
+    title: "Clickable / ClickableBehavior",
+    component: ClickableBehavior,
+    argTypes: argTypes,
+    args: {
+        disabled: false,
+    },
+    parameters: {
+        componentSubtitle: ((
+            <ComponentInfo name={name} version={version} />
+        ): any),
+        docs: {
+            description: {
+                component: null,
+            },
+            source: {
+                // See https://github.com/storybookjs/storybook/issues/12596
+                excludeDecorators: true,
+            },
+        },
+    },
+};
+
+export const Default: StoryComponentType = (args) => {
+    const ClickableBehavior = getClickableBehavior();
+
+    return (
+        <ClickableBehavior role="button" {...args}>
+            {(state, childrenProps) => {
+                const {pressed, hovered, focused} = state;
+                return (
+                    <View
+                        style={[
+                            styles.clickable,
+                            hovered && styles.hovered,
+                            focused && styles.focused,
+                            pressed && styles.pressed,
+                        ]}
+                        {...childrenProps}
+                    >
+                        This is an element wrapped with ClickableBehavior
+                    </View>
+                );
+            }}
+        </ClickableBehavior>
+    );
+};
+
+Default.parameters = {
+    chromatic: {
+        // we don't need screenshots because this story only displays the
+        // resting/default state.
+        disableSnapshot: true,
+    },
+};
+
+const styles = StyleSheet.create({
+    clickable: {
+        cursor: "pointer",
```

Comment:
It looks like the cursor remains `pointer` when using the controls to set this story in the disabled state.
Wha do you think about adding a `disabled` style and adding `disabled && styles.disabled` in the `View`'s style prop? 

Or do you think it would be better to leave that out to demonstrate the inherent behavior of `ClickableBehavior`? In that case, I'd argue that we shouldn't add the `cursor: "pointer"` style either.

Reply:
Good point, I agree that the latter is better, so I'm going to remove that cursor from the example.